cpp/velox/memory/VeloxMemoryPool.cc

```diff
@@ -106,25 +123,24 @@ class WrappedVeloxMemoryPool : public facebook::velox::memory::MemoryPool {
     if (this->isMemoryCapped()) {
       VELOX_MEM_MANUAL_CAP();
     }
-    auto alignedSize = sizeAlign<ALIGNMENT>(ALIGNER<ALIGNMENT>{}, size);
+    auto alignedSize = sizeAlign(size);
```

Comment:
@zhztheplayer can you take a look?

Aligned allocation should have better performance. It may also impact avx

Reply:
He asked for leave this afternoon. Maybe he'll check it tomorrow.

Reply:
Thanks and I just got some time :)

It looks like the only change from the code is to use a variable for the alignment width (default to 64 bytes) instead of using template. This looks fine to me.

Reply:
Yes. meta/Velox  use "options.alignment" variable to replace template ALIGNMENT.